lib/content-services/src/lib/aspect-list/aspect-list.component.spec.ts

```diff
@@ -250,6 +250,21 @@ describe('AspectListComponent', () => {
                 component.clear();
                 expect(component.nodeAspects.length).toBe(0);
             });
+
+            it('should store node aspects that are not listed and emit then on value change', async () => {
+                const storedAspect: string[] = ['stored:aspect'];
+
+                expect(component.notDisplayedAspects).toEqual(storedAspect);
+
+                spyOn(component.valueChanged, 'emit');
+                const panel = await loader.getAllHarnesses(MatExpansionPanelHarness);
+                await panel[0].expand();
+                const checkbox = await panel[0].getHarness(MatCheckboxHarness);
```

Comment:
you have twice panel[0] so could you please move it to variable to don't repeat searching of array? 

You can even resolve both comments at once like: 
const panel = await loader.getAllHarnesses(MatExpansionPanelHarness)[0];

then no need even for plural as you will store only single panel there.